src/crystal/system/win32/time.cr

```diff
@@ -1,9 +1,33 @@
+require "c/time"
+
 module Crystal::System::Time
+  WINDOWS_EPOCH_IN_SECONDS = 50491123200_i64
+
   def self.compute_utc_offset(seconds : Int64) : Int32
-    raise NotImplementedError.new("Crystal::System::Time.compute_utc_offset")
+    ret = LibC.get_time_zone_information(out tz)
+    raise Errno.new("GetTimeZoneInformation") if ret == -1
+
+    tz.bias.to_i32 * -60
   end
 
   def self.compute_utc_seconds_and_nanoseconds : {Int64, Int32}
-    raise NotImplementedError.new("Crystal::System::Time.compute_utc_seconds_and_nanoseconds")
+    LibC.get_system_time_as_file_time(out filetime)
+    since_epoch = (filetime.high_date_time.to_u64 << 32) | filetime.low_date_time.to_u64
+
+    {(since_epoch / 10_000_000).to_i64 + WINDOWS_EPOCH_IN_SECONDS, since_epoch.remainder(10_000_000).to_i32 * 100}
```

Comment:
would be nice to name these constants: `FILETIME_TICKS_PER_SECOND` and `NANOSECONDS_PER_FILETIME_TICK`

Reply:
`FILETIME` does not really represent ticks but 100 nanosecond intervals. But I guess it's fine as a name and I don't have any better idea.